tests/snuba/api/endpoints/test_organization_events.py

```diff
@@ -778,6 +778,157 @@ def test_with_event_count_flag(self):
         ]
 
 
+class OrganizationEventsTagsEndpointTest(OrganizationEventsTestBase):
+    def setUp(self):
+        super(OrganizationEventsTagsEndpointTest, self).setUp()
+        self.login_as(user=self.user)
+        self.project = self.create_project()
+        self.group = self.create_group(project=self.project)
+        self.url = reverse(
+            'sentry-api-0-organization-events-tags',
+            kwargs={
+                'organization_slug': self.project.organization.slug,
+                'key': 'color',
+            }
+        )
+
+    def assert_top_values(self, output, expected):
+        assert len(output) == len(expected)
+        for index, (count, value) in enumerate(expected):
+            assert output[index]['count'] == count
+            assert output[index]['value'] == value
+
+    def test_simple(self):
+        self.create_event(
```

Comment:
You should use `store_event` instead of `create_event` there are docstrings on `create_event` that it is deprecated.

Keep in mind that `store_event` will require you to change your event_id values to be valid hexadecimal characters.

Reply:
@markstory  I think this whole file uses `create_event` but I'll change it for the changes I've made in this pr.